### PR TITLE
Added ebuilds for Pale Moon 29.4.1

### PR DESCRIPTION
**Pull request by eslafgh**
The Pale Moon developers have restricted access to the development git repository and so for versions 29.4.1 source snaps shots are going to be the only way to retrieve the source code, see forum post [Attention package builders! 29.4.1 and later](https://forum.palemoon.org/viewtopic.php?f=5&t=27369).

This ebuild uses the source snap shot to build palemoon 29.4.1.

### CODE REVIEW

File: www-client/palemoon/palemoon-29.4.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+"
+
+REQUIRED_USE="
+	optimize? ( !debug )
+	jemalloc? ( !valgrind )
+	^^ ( gtk2 gtk3 )
+	necko-wifi? ( dbus )
+"
+
+src_unpack() {
```
Comment (CaseOf) [inline www-client/palemoon/palemoon-29.4.1.ebuild:76]: I dont think this is needed, you can just add S=${WORKDIR} to make it work.

File: www-client/palemoon/palemoon-29.4.1.ebuild [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+REQUIRED_BUILDSPACE='16G'
+GCC_SUPPORTED_VERSIONS="7.5 8.5 9.3 9.4 10.3"
+
+inherit palemoon-5 eutils flag-o-matic pax-utils
+
+KEYWORDS="~x86 amd64"
+DESCRIPTION="Pale Moon Web Browser"
+HOMEPAGE="https://www.palemoon.org/"
+
+SRC_URI="http://archive.palemoon.org/source/palemoon-${PV}-source.tar.xz"
```
Comment (CaseOf) [inline www-client/palemoon/palemoon-29.4.1.ebuild:12]: Not your fault here, but I’m not really fan of http only archive distribution.
I’ll try to convince @MoonchildProductions to add at least checksums for each archive release.

Comment (CaseOf) [reply www-client/palemoon/palemoon-29.4.1.ebuild:12]: you can use xz -t file.tar.xz to check integrity
I did not know this feature existed.

File: www-client/palemoon/palemoon-29.4.1.ebuild (hunk 1 above)
Comment (eslafgh) [reply www-client/palemoon/palemoon-29.4.1.ebuild:76]: Did not think of that, will update.

**Comment by eslafgh** [comment]
Updated to remove unpack as suggested.
Also added palemoon-bin ebuild for 29.4.1.

**Comment by eslafgh** [comment]
Don't try to download from mirrors, i.e. added RESTRICT="mirror"

File: www-client/palemoon/palemoon-29.4.1.ebuild (hunk 2 above)
Comment (jeandestouches) [reply www-client/palemoon/palemoon-29.4.1.ebuild:13]: Hi CaseOf, is there some news regarding checksum / sign for the archive release ? I don't have an account on palemoon forums to ask. Hopefully, the next release will have it.
Cheers

Comment (CaseOf) [reply www-client/palemoon/palemoon-29.4.1.ebuild:13]: Hi @jean, I did talk with @MoonChildPM and he did told me is that xz actually the ability to test its integrity. We can use that for now.

Comment (jeandestouches) [reply www-client/palemoon/palemoon-29.4.1.ebuild:13]: @CaseOf Got it, thanks for your answer.

**Pull request merged**